dtool/PandaVersion.pp

```diff
@@ -48,4 +48,4 @@
 // integers, but by convention it will consist of four integers, with
 // the first three matching the plugin version, and the fourth integer
 // being incremented with each new Core API revision.
-#define P3D_COREAPI_VERSION $[P3D_PLUGIN_VERSION] 2
+#define P3D_COREAPI_VERSION
```

Comment:
What's the rationale for not removing these lines altogether?

I think the only thing using this is `ParsePluginVersion` in `makepandacore.py`, which is used by `makepanda/getversion.py --runtime`, but its `--runtime` handling should be removed, so then we can also remove `ParsePluginVersion` and `ParseCoreapiVersion` from `makepandacore.py`.

Reply:
I was thinking it was used by PandaSystem, but it's not so I'l remove them.

Reply:
We can change the methods on PandaSystem to do whatever they do if the symbols aren't defined.  We should also mark those methods as deprecated.

Reply:
How do we deprecate C++ code?

Reply:
At the very least put a `@deprecated` tag in the docstring like so:
```cpp
/**
 * Method that does something.
 *
 * @deprecated The runtime system has been removed, this now does nothing.
 */
void PandaSystem::
do_something() {
```

Adding a warning using `std::cerr << "Using deprecated PandaSystem.do_something()\n";` under an `#ifndef NDEBUG` would be appropriate if we want to discourage people even more strongly.  (The prc/logging stuff hasn't been defined yet in dtoolbase, so we can't rely on `nassert_raise` et al.)

Reply:
I have the `@deprecated` tag in the docstrings. I can also add the `cerr` prints if you'd like.